features/filmStrip/components/FilmStrip.js

```diff
@@ -40,6 +40,10 @@ class FilmStrip extends Component {
                                     participant={p}
                                     videoTrack={videoTrack}
                                     audioTrack={audioTrack}
+                                    audioMuted={!audioTrack ||
```

Comment:
Well, that is an example of a situation in which a source code comment is called for.